tests/expect/composer_paths_01.exp

```diff
@@ -9,9 +12,24 @@ expect "::debug::Composer path is '*'\r
 ::debug::Composer cache directory found at '*'\r
 ::debug::File composer.json found at './composer.json'\r
 ::debug::File composer.lock path computed as './composer.lock'\r
-::set-output name=command::*\r
-::set-output name=cache-dir::*\r
-::set-output name=json::./composer.json\r
-::set-output name=lock::./composer.lock\r
 "
 expect eof
+
+set fp [open $gitHubOutputFile r]
+set fileData [read $fp]
+close $fp
+
+set expectedValue "command=*\r
+cache-dir=*\r
+json=./composer.json\r
+lock=./composer.lock\n
+"
+
+if { $expectedValue != $fileData } {
```

Comment:
This test fails as it needs to do a regex compare, not a direct comparison.

Reply:
My bash scripting is admittedly poor. Do you have an example I could use as a guide?

Reply:
My bash scripting skills are even worse, then again, my Google skills are reasonable ;-)

* Looks like a [wildcard compare is possible](https://stackoverflow.com/a/19897118), but the pattern used here is probably too complex for that.
* This [guide about pattern matching and regex in bash](https://riptutorial.com/bash/topic/3795/pattern-matching-and-regular-expressions) looks very helpful.
* And [this one](https://www.shell-tips.com/bash/wildcards-globbing/#gsc.tab=0) may be of use as well.